pol-core/pol/item/weapon.cpp

```diff
@@ -458,7 +458,7 @@ bool UWeapon::consume_projectile( Core::UContainer* cont ) const
 
 bool UWeapon::in_range( const Mobile::Character* wielder, const Mobile::Character* target ) const
 {
-  unsigned short dist = pol_distance( wielder, target );
+  unsigned short dist = wielder->pos().pol_distance( target->pos() );
```

Comment:
what do you think about `wielder->distance_to(...)`?

Reply:
#581